feeder/src/vote.ts

```diff
@@ -65,13 +62,11 @@ async function loadOracleParams(client: LCDClient): Promise<{
 }
 
 interface Price {
-  currency: string
+  denom: string
   price: string
 }
 
 async function getPrices(sources: string[]): Promise<Price[]> {
-  console.info(`getPrices: source: ${sources.join(',')}`)
-
```

Comment:
Is there a reason why we use bluebird promises instead of native promises?

Reply:
The main reasons to use bluebird is [some](http://bluebirdjs.com/docs/api/promise.some.html#promise.some) and [mapSeries](http://bluebirdjs.com/docs/api/promise.mapseries.html). The two mentioned features are not available in native Promise and they have a very important impact on optimization.